tests/lib/prepare-restore.sh

```diff
@@ -75,7 +75,8 @@ build_deb(){
         rm -rf vendor/*/*
     fi
 
-    su -l -c "cd $PWD && DEB_BUILD_OPTIONS='nocheck testkeys' dpkg-buildpackage -tc -b -Zgzip" test
+    unshare -n \
```

Comment:
yeah I think this needs to be `unshare -n -- su ...`

Reply:
Thanks for spotting. Heh ofc it failed on 14.04, where else it would fail like this.